### DISCUSSION TITLE
Question 1: Extra records in group [ZK - Action]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#240] [Q&A] [answered]
Description: Group IRN 10678 shows that it has 163 catalogue records, but when I open it up it only displays 155. There should only be 155 records in this group, so that is correct but I am wondering why the group has 163 records on the static tab?

**Comment by rondlg** [comment]
Response: Try refreshing the group and if the same behaviour occurs then send a helpdesk ticket.

**Comment by fmjjones** [comment, marked as answer]
Axiell's response is that the Groups Module is designed to allow a single record to be attached multiple times. In order to prevent that same thing from happening again, use the Group tool from the module (in your case Catalogue Module) that you want to retrieve the records from. There is a validation layer on the tool that rejects the duplicate entries.
